shared-module/tilepalettemapper/TilePaletteMapper.c

```diff
@@ -67,7 +55,7 @@ void common_hal_tilepalettemapper_tilepalettemapper_set_mapping(tilepalettemappe
         mp_arg_validate_int_range(mapping_val, 0, palette_max, MP_QSTR_mapping_value);
         self->tile_mappings[y * self->width_in_tiles + x][i] = mapping_val;
     }
-    self->needs_refresh = true;
+    displayio_tilegrid_mark_tile_dirty(self->tilegrid, x, y);
```

Comment:
Is there a chance that someone will set the existing mapping again? Maybe we want to detect when nothing is actually changed and not mark the tile dirty then.